_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -152,7 +152,7 @@ This off screen `img` element has an [implicit role][] of `presentation` because
 
 #### Failed Example 1
 
-This `img` element has an empty [accessible name][] and an [implicit role][] of `img` because it is missing an empty `alt` attribute.
+This `img` element has an empty [accessible name][] and an [implicit role][] of `img` because it is missing an attribute to give it an [accessible name][].
```

Comment:
```suggestion
This `img` element has an [implicit role][] of `img` and an empty [accessible name][] because it is missing an attribute to give it an [accessible name][].
```
Keeping the "because" clause next to the text it explains.